Makefile

```diff
@@ -119,6 +119,13 @@ else
 KUSTOMIZE=$(shell which kustomize)
 endif
 
+olm-catalog: manifests
+	operator-sdk generate kustomize manifests -q
+	cd config/manager && $(KUSTOMIZE) edit set image controller=REPLACE_IMAGE:TAG
+	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS) --input-dir bundle
+	./hack/update-csv.py bundle/manifests/ssp-operator.clusterserviceversion.yaml > config/olm-catalog/ssp-operator.clusterserviceversion.yaml
```

Comment:
I think it would be better if the resulting `.yaml` file is not in the `config` directory. Maybe we can but it to `data` directory since it has to be copied to the container image.

I would also put it to `.gitignore` and generate it on `make container-build`.